packages/apputils-extension/src/index.ts

```diff
@@ -606,6 +608,14 @@ const utilityCommands: JupyterFrontEndPlugin<void> = {
         }
       }
     });
+
+    commands.addCommand(CommandIDs.displayShortcuts, {
```

Comment:
Could you add a shortcut for the new command?

You need to create a new file `packages/apputils-extension/schema/utilityCommands.json` that contains:

```json
{
  "title": "Shortcuts Help",
  "description": "Shortcut help settings.",
  "jupyter.lab.shortcuts": [
    {
      "command": "apputils:display-shortcuts",
      "keys": ["Accel Shift H"],
      "selector": "body"
    }
  ],
  "properties": {},
  "type": "object",
  "additionalProperties": false
}
```

Reply:
This has been added.

Reply:
@jtpio Thanks for the suggestion. I will try to add the command to the main menu and to the command palette.

Reply:
Question: did we discuss the shortcut? In classic notebook it was just <kbd>H</kbd> in command mode.

For context, this is where it is in classic notebook:

![Screenshot from 2023-02-23 20-20-01](https://user-images.githubusercontent.com/5832902/221021003-833a1eb0-ddf6-438d-aaf0-c349bad96d54.png)

Reply:
@krassowski You suggest to use capital H too for this shortcut in Jupyter Lab ? I don't have an opinion on the choice. @fcollonval What do you think about it ?

Reply:
@krassowski I fear using `H` is to simple to be robust in JupyterLab. And as the shortcuts help can be displayed on any document, it will for example be impossible to use it with the file editor. So we would end up having multiple shortcuts depending on the active document.

But at least the infrastructure does not prevent us to use `H` in Notebook v7.